com.unity.shadergraph/Editor/Generation/TargetResources/BlockFields.cs

```diff
@@ -15,6 +15,12 @@ public struct VertexDescription
                 new NormalControl(CoordinateSpace.Object), ShaderStage.Vertex);
             public static BlockFieldDescriptor Tangent       = new BlockFieldDescriptor(VertexDescription.name, "Tangent", "VERTEXDESCRIPTION_TANGENT",
                 new TangentControl(CoordinateSpace.Object), ShaderStage.Vertex);
+
+            // TODO: In the future we may have proper ShaderStage for tessellation, this will need to be revisit then
+            public static BlockFieldDescriptor TessellationFactor = new BlockFieldDescriptor(SurfaceDescription.name, "TessellationFactor", "Tessellation Factor", "VERTEXDESCRIPTION_TESSELLATIONFACTOR",
+                new FloatControl(1.0f), ShaderStage.Vertex);
+            public static BlockFieldDescriptor TessellationDisplacement = new BlockFieldDescriptor(SurfaceDescription.name, "TessellationDisplacement", "Tessellation Displacement", "VERTEXDESCRIPTION_TESSELLATIONDISPLACEMENT",
+                new PositionControl(CoordinateSpace.World), ShaderStage.Vertex);
```

Comment:
Should probably be VertexDescription.name instead of SurfaceDescription.name here.

Reply:
alright, will do the change.